src/interactive-window/commands/commandRegistry.ts

```diff
@@ -55,28 +38,12 @@ export class CommandRegistry implements IDisposable, IExtensionSingleActivationS
         @inject(IDebugService) @optional() private debugService: IDebugService | undefined,
         @inject(IConfigurationService) private configService: IConfigurationService,
         @inject(IApplicationShell) private appShell: IApplicationShell,
-        @inject(IExportCommands) @optional() private readonly exportCommand: IExportCommands | undefined,
```

Comment:
Export is universal, whether the document is notebook or iw is irrelevant so here we move it out of iw.